ui/src/app/pages/admin/user/admin-user-egress-bypass.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
```

Comment:
please expand this to cover the logic of the component

Reply:
Since its a new file, i would prefer to use react testing libraries. So will it be ok if that is a different PR?